app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,74 @@
+class PaymentsController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+  load_and_authorize_resource
```

Comment:
**Optional**
- Please delete the repeated line of code here for a cleaner code base